src/locale/localed-util.h

```diff
@@ -26,16 +26,12 @@ typedef struct Context {
 } Context;
 
 int x11_context_verify_and_warn(const X11Context *xc, int log_level, sd_bus_error *error);
-static inline int x11_context_verify(const X11Context *xc) {
-        return x11_context_verify_and_warn(xc, LOG_DEBUG, NULL);
-}
+int x11_context_verify(const X11Context *xc);
 
 X11Context *context_get_x11_context(Context *c);
 
 int vc_context_verify_and_warn(const VCContext *vc, int log_level, sd_bus_error *error);
-static inline int vc_context_verify(const VCContext *vc) {
-        return vc_context_verify_and_warn(vc, LOG_DEBUG, NULL);
-}
+int vc_context_verify(const VCContext *vc);
```

Comment:
I do not think the change is necessary. I guess the change is made for dropping LOG_DEBUG from the header to avoid log.h from the header. But, you can get LOG_DEBUG by including syslog.h.